x-pack/plugin/core/src/main/resources/security-index-template-7.json

```diff
@@ -1,5 +1,5 @@
 {
-  "index_patterns" : [ ".security-*" ],
```

Comment:
The `index_patterns` are not used because the security index is not managed using templates anymore. The change in name is solely for doc purposes.